src/openforms/registrations/contrib/zgw_apis/plugin.py

```diff
@@ -444,3 +490,61 @@ def get_config_actions(self) -> list[Action]:
                 ),
             ),
         ]
+
+    def register_submission_to_objects_api(
```

Comment:
This could be a function in the Objects API itself (in `utils.py` or in `client.py)` and used by both ZGW and Objects API. I thought that it is more clear to have it here duplicated and show exactly what we are doing but feel free to tell me if this should be in another file and used by both.

Reply:
Taking a look at the `ObjectsAPIRegistration.register_submission`, most of it seems to be duplicated, but at the same time the context doesn't seem to be the same. So trying to be DRY here might end up being more confusing. Perhaps having a single method handling things starting at `record_data = ...`?